src/staging/src/hooks/useContent.js

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import * as cheerio from "cheerio";
+
+export const useContent = (contentHtml) => {
+  const [title, setTitle] = useState("");
+  let video_id;
+  let media; 
+  let getIframe
+
+  if(contentHtml?.introHtml){
+    let $ =  cheerio.load(contentHtml?.introHtml);
+    getIframe = $("iframe");
```

Comment:
I'm not sure if there are parks pages with multiple videos, but this selector could return more than 1 iframe.  
jQuery would return multiple matches if your selector had multiple matches.  It looks like Cheerio has a toArray() method that behaves the same as jQuery.  
https://www.tabnine.com/code/javascript/functions/cheerio/Cheerio/toArray